### ISSUE TITLE
Mobile pages not displayed on browser refresh FF 

### ISSUE THREAD

**Issue by billreed63** [maqetta/maqetta#3346]
If I have two mobile pages open in tab editors
in tab 1 is file1.html and in tab 2 is file2.html  with tab 2 selected refresh FF browser
After browser refresh select tab one

I see a blank screen in tab 1 with just the background color set.

**Comment by billreed63** [comment]
file1.html

```
<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8"/>
<title>Untitled</title>
<script type="text/javascript" src="lib/dojo/dojo/dojo.js" data-dojo-config="'parseOnLoad':true,'async':true,'packages':[{'name':'gridx','location':'../gridx'},{'name':'idx','location':'../../idx'},{'name':'idxThemes','location':'../../../themes'},{'name':'clipart','location':'../../clipart'},{'name':'maqettaSamples','location':'../../../samples'},{'name':'maqetta','location':'../../maqetta'},{'name':'shapes','location':'../../shapes'},{'name':'widgets','location':'../../custom'}],'themeMap':[['Android','',['themes/custom/custom.css']],['BlackBerry','',['themes/custom/custom.css']],['iPad','',['themes/custom/custom.css']],['iPhone','',['themes/custom/custom.css']],['.*','',['themes/custom/custom.css']]],'mblThemeFiles':[], mblLoadCompatPattern: /\/project1\/themes\/.*\.css$/"></script>
<script type="text/javascript">
require([
  "dijit/dijit",
  "dojo/parser",
  "maqetta/space",
  "maqetta/AppStates",
  "dojox/mobile",
  "dojox/mobile/deviceTheme",
  "dojox/mobile/compat",
  "dojo/hash"
]);
</script>
<style>@import "themes/claro/claro.css";@import "app.css";
</style>
<script type="text/javascript" src="app.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no"></meta>
</head>
<body class="claro" data-maq-device="iphone" data-maq-flow-layout="true" data-maq-ws="collapse" id="myapp" data-maq-appstates="{}">
 <div id="View_0" data-dojo-type="dojox.mobile.View" style="height: 100%;" selected="selected">
  <h1 label="Heading" data-dojo-type="dojox.mobile.Heading"></h1>
</div>
</body>
</html>
```

file2.html

```
<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8"/>
<title>Untitled</title>
<script type="text/javascript" src="lib/dojo/dojo/dojo.js" data-dojo-config="'parseOnLoad':true,'async':true,'packages':[{'name':'gridx','location':'../gridx'},{'name':'idx','location':'../../idx'},{'name':'idxThemes','location':'../../../themes'},{'name':'clipart','location':'../../clipart'},{'name':'maqettaSamples','location':'../../../samples'},{'name':'maqetta','location':'../../maqetta'},{'name':'shapes','location':'../../shapes'},{'name':'widgets','location':'../../custom'}]"></script>
<script type="text/javascript">
require([
  "dijit/dijit",
  "dojo/parser",
  "maqetta/space",
  "maqetta/AppStates",
  "dojox/mobile",
  "dojox/mobile/deviceTheme",
  "dojox/mobile/compat",
  "dojo/hash"
]);
</script>
<style>@import "themes/claro/claro.css";@import "app.css";
</style>
<script type="text/javascript" src="app.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no"></meta>
</head>
<body class="claro" data-maq-device="iphone" data-maq-flow-layout="true" data-maq-ws="collapse" id="myapp" data-maq-appstates="{}">
 <div id="View_0" data-dojo-type="dojox.mobile.View" style="height: 100%;" selected="selected">
  <h1 label="Heading" data-dojo-type="dojox.mobile.Heading"></h1>
</div>
</body>
</html>
```

**Comment by JonFerraiolo** [comment]
I've seen this problem intermittently for months, but @billreed63 has a reproducible scenario.

**Comment by peller** [comment]
yes, likely a dup of #3279

**Comment by jhpedemonte** [comment]
-> M8

**Comment by JonFerraiolo** [comment]
Since #3279 is 7.0.1., I'm moving this also to 7.0.1.

**Comment by JonFerraiolo** [comment]
oops, didn't mean to close

**Comment by peller** [comment]
confirmed working now on master and 701 branch

**Comment by JonFerraiolo** [comment]
Reopening. I'm seeing the problem in today's build on the testing server.

**Comment by JonFerraiolo** [comment]
@jhpedemonte 

The fix for #3279 this didn't get properly cherry-picked into the M7_0_1 branch.

**Comment by JonFerraiolo** [comment]
I just cherry-picked and pushed the fix for #3279 into M7_0_1 branch. Let's retest when a new build is available.

**Comment by JonFerraiolo** [comment]
Closing again. Appears to be fixed in M7_0_1 builds.
